packages/react-core/src/components/Nav/NavExpandable.tsx

```diff
@@ -90,9 +93,27 @@ export class NavExpandable extends React.Component<NavExpandableProps, NavExpand
   };
 
   render() {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const { id, title, srText, children, className, isActive, groupId, isExpanded, onExpand, ...props } = this.props;
+    const {
+      onToggle: onToggleProp,
```

Comment:
You do not need to rename this here.  I would change the name of the function on line 113 instead.  Maybe onClick?  Then you can also get rid of line 111 and declare the function on line 113 as a const.

Reply:
good point! will fix.